tests/pytorch/test_pytorch_model_export.py

```diff
@@ -738,6 +758,40 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+@pytest.mark.large
+@pytest.mark.parametrize("scripted_model", [True, False])
+def test_load_model_succeeds_when_data_is_model_file_instead_of_directory_saved_as_state_dict(
+    sequential_model, model_path, data
+):
+    """
+    This test verifies that PyTorch models saved in older versions of MLflow are loaded successfully
+    by ``mlflow.pytorch.load_model``. The ``data`` path associated with these older models is
+    serialized PyTorch model file, as opposed to the current format: a directory containing a
+    serialized model file and pickle module information.
+    """
+    artifact_path = "pytorch_model"
+    with mlflow.start_run():
+        mlflow.pytorch.log_state_dict(
+            artifact_path=artifact_path, state_dict=sequential_model.state_dict()
```

Comment:
```suggestion
            state_dict=sequential_model.state_dict(), artifact_path=artifact_path
```

This is a nit but can we swap the arguments here?

Reply:
Okies. Swapped the arguments.